lib/qiita/markdown/filters/external_link.rb

```diff
@@ -0,0 +1,35 @@
+module Qiita
+  module Markdown
+    module Filters
+      class ExternalLink < HTML::Pipeline::Filter
+        def call
+          return doc unless hostname
```

Comment:
> should we NOT add the ExternalLink to the DEFAULT_FILTERS defined at lib/qiita/markdown/processor.rb so that all qiita-markdown users can continue to use qiita-markdown without specifying hostname parameter?

I think we should add `ExternalLink` filter to the default filters. I understand your concern about the backward compatibility, but I think the default behavior of qiita-markdown should reflect the current Qiita implementation.

Because we are still in the [initial development phase in SemVer](http://semver.org/#spec-item-4) and I don't think this gem is not widely used, we can introduce breakage if needed 😉